app/assets/stylesheets/layout_components/siteheader.scss

```diff
@@ -95,15 +95,17 @@
     padding: 0 $base-padding;
   }
 
+  a{
```

Comment:
This is how you'd add a class to the link here https://github.com/amaabca/ama_layout/blob/6c53de84f6a6cb59ce0991b939e161cc5b2018fb/app/views/ama_layout/_siteheader.html.erb#L4

`<%= link_to(image_tag("ama-logo.png", class: "title-bar__logo"), "https://www.ama.ab.ca", class: "waffle-time") %>`